### PR TITLE
fix: add missing `preset-wind` to typescript config file

### PR DESCRIPTION
**Pull request by userquin**
This PR fix the build problem on windows

### CODE REVIEW

**Review by ydcjeff — APPROVED** [review]

File: packages/preset-wind/build.config.ts [hunk 1]
```diff
@@ -6,6 +6,9 @@ export default defineBuildConfig({
   ],
   clean: true,
   declaration: true,
+  externals: [
+    'unconfig',
+  ],
```
Comment (hannoeru) [inline packages/preset-wind/build.config.ts:11]: Why do we need externals unconfig here?

Comment (userquin) [reply packages/preset-wind/build.config.ts:11]: <details>
<summary>for this, jut check the <strong>error</strong></summary>

```shell
 Error building F:/work/projects/quini/GitHub/antfu/unocss-main-clean/packages/preset-wind: SyntaxError: Unexpected token (1:7) in F:\work\projects\quini\GitHub\antfu\unocss-main-clean\node_modules\.pnpm\jiti  
│ SyntaxError: Unexpected token (1:7) in F:\work\projects\quini\GitHub\antfu\unocss-main-clean\node_modules\.pnpm\jiti@1.12.15\node_modules\jiti\dist\types.d.ts
│     at Parser.pp$4.raise (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:19477:13)
│     at Parser.pp$9.unexpected (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:16773:8)
│     at Parser.pp$9.expect (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:16767:26)
│     at Parser.pp$8.parseExportSpecifiers (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:17786:8)
│     at Parser.pp$8.parseExport (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:17703:28)
│     at Parser.pp$8.parseStatement (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:16949:74)
│     at Parser.pp$8.parseTopLevel (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:16830:21)
│     at Parser.parse (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:16603:15)
│     at Function.parse (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:16653:35)
│     at Graph.contextParse (file:///F:/work/projects/quini/GitHub/antfu/unocss-main-clean/node_modules/.pnpm/rollup@2.68.0/node_modules/rollup/dist/es/shared/rollup.js:22817:38) {
│   pos: 7,
│   loc: {
│     column: 7,
│     file: 'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\types.d.ts',
│     line: 1
│   },
│   raisedAt: 14,
│   frame: '1: export declare type TransformOptions = {\n' +
│     '          ^\n' +
│     '2:     source: string;\n' +
│     '3:     filename?: string;',
│   id: 'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\types.d.ts',
│   hook: 'transform',
│   code: 'PLUGIN_ERROR',
│   plugin: 'commonjs',
│   watchFiles: [
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\theme.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\shortcuts.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\behaviors.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\background.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\columns.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\animation.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\filters.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\default.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\container.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\divide.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\scrolls.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\spacing.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\line-clamp.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\static.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\placeholder.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\touch-actions.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\table.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\typography.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\rules\\variables.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\dark.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\media.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\combinators.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\default.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\placeholder.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\preset-wind\\src\\variants\\misc.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\types.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\extractors\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\generator\\index.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\extractors\\svelte.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\extractors\\split.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\unconfig@0.3.1_@vitest+ui@0.5.1\\node_modules\\unconfig\\dist\\index.mjs',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\basic.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\object.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\helpers.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\escape.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\map.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\variantGroup.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\layer.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\handlers.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\packages\\core\\src\\utils\\warn.ts',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\defu@5.0.1\\node_modules\\defu\\dist\\defu.mjs',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\@antfu+utils@0.5.0\\node_modules\\@antfu\\utils\\dist\\index.mjs',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\lib\\index.js',
│     '\x00commonjsHelpers.js',
│     '\x00F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\jiti.js?commonjs-proxy',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\jiti.js',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\babel.js',
│     '\x00F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\babel.js?commonjs-proxy',
│     'F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\types.d.ts'
│   ]
│ }
│ [!] Error: unfinished hook action(s) on exit:
│ (node-resolve) resolveId "F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\jiti.js" "\u0000F:\\work\\projects\\quini\\GitHub\\antfu\\u  
│ (node-resolve) resolveId "F:\\work\\projects\\quini\\GitHub\\antfu\\unocss-main-clean\\node_modules\\.pnpm\\jiti@1.12.15\\node_modules\\jiti\\dist\\babel.js" "\u0000F:\\work\\projects\\quini\\GitHub\\antfu\\  
└─ Failed in 8.6s
packages/vite build$ unbuild
[59 lines collapsed]
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\index.ts
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\index.ts
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\index.ts
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\index.ts
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\index.ts
│ Entry module "src\index.ts" is using named and default exports together. Consumers of your bundle will have to use `chunk["default"]` to access the default export, which may not be what you want. Use `output  
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\index.ts
│ √ Build succeed for @unocss/vite
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\defaults.ts
│ Entry module "src\index.ts" is using named and default exports together. Consumers of your bundle will have to use `chunk["default"]` to access the default export, which may not be what you want. Use `output  
│  WARN  Inlining implicit external F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\plugins-common\defaults.ts (repeated 2 times)
│ √ Build succeed for @unocss/webpack
│   dist/index.cjs (size: 24 kB, exports: default, defineConfig)
│   dist/index.mjs (size: 22.7 kB, exports: default, defineConfig)
└─ Done in 4s
F:\work\projects\quini\GitHub\antfu\unocss-main-clean\packages\preset-wind:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @unocss/preset-wind@0.26.4 build: `unbuild`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.
```
</details>

Comment (userquin) [reply packages/preset-wind/build.config.ts:11]: @hannoeru it seems the error only on windows, on macos for example the build works with and without this PR.

Comment (hannoeru) [reply packages/preset-wind/build.config.ts:11]: @userquin I had this issue before, cleanup dist folder can fix that.

Comment (userquin) [reply packages/preset-wind/build.config.ts:11]: @hannoeru the `dist` folder is cleared, Anthony just add the `rimraf packages/*/dist` to the build script

**Comment by userquin** [comment]
it is also reported on `jiti` (https://github.com/unjs/jiti/issues/49)  and some similar issue on `unbuild` (https://github.com/unjs/unbuild/issues/38)

**Comment by userquin** [comment]
it seems removing from `unbuild::rollup.mjs` just works: `rollupOptions.plugins.push(dts(/*{ respectExternal: true }*/))`

imagen

**Pull request merged**